ocean.tf

```diff
@@ -164,6 +164,31 @@ resource "spotinst_ocean_aws" "ocean" {
       }
     }
   }
+  ## Block Device Mappings ##
+  dynamic "block_device_mappings" {
+    for_each = var.block_device_mappings
```

Comment:
add the below logic
!= null ? [var.block_device_mappings] : []

Reply:
We will get error message, if we use above suggested logic.

Error: Unsupported attribute
│ 
│   on ../../ocean.tf line 171, in resource "spotinst_ocean_aws" "ocean":
│  171:       device_name = block_device_mappings.value.device_name
│     ├────────────────
│     │ block_device_mappings.value is list of object with 1 element
│ 
│ Can't access attributes on a list of objects. Did you mean to access attribute "device_name" for a specific element of the list, or across all elements of the list?